play/src/components/GameSessionSwitch.tsx

```diff
@@ -35,12 +35,12 @@ export default function GameSessionSwitch({
   const { currentState } = gameSession;
   const currentQuestion =
     gameSession.questions[gameSession.currentQuestionIndex!]; // eslint-disable-line @typescript-eslint/no-non-null-assertion
-  const currentTeam = gameSession.teams!.find( // eslint-disable-line @typescript-eslint/no-non-null-assertion
+  const currentTeam = gameSession.teams!.find(
+    // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
do you mind bringing this // eslint-disable-line back to the original place? This will keep it from showing up in this PR